2023-10-07 16:23:56,830 INFO [streaming_server.py:822] {'encoder': './sherpa-onnx-streaming-zipformer-en-2023-06-21/encoder-epoch-99-avg-1.int8.onnx', 'decoder': './sherpa-onnx-streaming-zipformer-en-2023-06-21/decoder-epoch-99-avg-1.int8.onnx', 'joiner': './sherpa-onnx-streaming-zipformer-en-2023-06-21/joiner-epoch-99-avg-1.int8.onnx', 'paraformer_encoder': None, 'paraformer_decoder': None, 'tokens': './sherpa-onnx-streaming-zipformer-en-2023-06-21/tokens.txt', 'sample_rate': 16000, 'feat_dim': 80, 'provider': 'cpu', 'decoding_method': 'greedy_search', 'num_active_paths': 4, 'use_endpoint': 1, 'rule1_min_trailing_silence': 2.4, 'rule2_min_trailing_silence': 1.2, 'rule3_min_utterance_length': 20, 'hotwords_file': '', 'hotwords_score': 1.5, 'port': 6006, 'nn_pool_size': 1, 'max_batch_size': 50, 'max_wait_ms': 10, 'max_message_size': 1048576, 'max_queue_size': 32, 'max_active_connections': 500, 'num_threads': 2, 'certificate': None, 'doc_root': 'web'}
2023-10-07 16:23:58,874 INFO [streaming_server.py:628] No certificate provided
2023-10-07 16:23:58,875 INFO [server.py:711] server listening on [::]:6006
2023-10-07 16:23:58,876 INFO [server.py:711] server listening on 0.0.0.0:6006
2023-10-07 16:23:58,876 INFO [streaming_server.py:654] Please visit one of the following addresses:

  http://localhost:6006

Since you are not providing a certificate, you cannot use your microphone from within the browser using public IP addresses. Only localhost can be used.You also cannot use 0.0.0.0 or 127.0.0.1
2023-10-07 16:24:05,893 INFO [server.py:233] connection failed (200 OK)
2023-10-07 16:24:05,894 INFO [server.py:268] connection closed
2023-10-07 16:24:05,972 INFO [server.py:233] connection failed (200 OK)
2023-10-07 16:24:05,973 INFO [server.py:268] connection closed
2023-10-07 16:24:05,997 INFO [server.py:233] connection failed (200 OK)
2023-10-07 16:24:05,998 INFO [server.py:233] connection failed (200 OK)
2023-10-07 16:24:05,998 INFO [server.py:233] connection failed (200 OK)
2023-10-07 16:24:05,999 INFO [server.py:233] connection failed (200 OK)
2023-10-07 16:24:05,999 INFO [server.py:268] connection closed
2023-10-07 16:24:05,999 INFO [server.py:268] connection closed
2023-10-07 16:24:05,999 INFO [server.py:268] connection closed
2023-10-07 16:24:05,999 INFO [server.py:268] connection closed
2023-10-07 16:24:06,059 INFO [server.py:233] connection failed (200 OK)
2023-10-07 16:24:06,060 INFO [server.py:268] connection closed
2023-10-07 16:24:07,247 INFO [server.py:646] connection open
2023-10-07 16:24:07,248 INFO [streaming_server.py:695] Connected: ('::1', 53418, 0, 0). Number of connections: 1/500
2023-10-07 16:24:08,026 INFO [streaming_server.py:679] Disconnected: ('::1', 53418, 0, 0). Number of connections: 0/500
2023-10-07 16:24:08,027 ERROR [server.py:242] connection handler failed
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/websockets/legacy/server.py", line 240, in handler
    await self.ws_handler(self)
  File "/Users/fawazahamedshaik/github/ASR/Interactive_ASR/./streaming_server.py", line 672, in handle_connection
    await self.handle_connection_impl(socket)
  File "/Users/fawazahamedshaik/github/ASR/Interactive_ASR/./streaming_server.py", line 746, in handle_connection_impl
    await socket.send(json.dumps(message))
                                 ^^^^^^^
UnboundLocalError: cannot access local variable 'message' where it is not associated with a value
2023-10-07 16:24:08,033 INFO [server.py:268] connection closed
2023-10-07 16:24:08,185 INFO [server.py:233] connection failed (200 OK)
2023-10-07 16:24:08,187 INFO [server.py:233] connection failed (200 OK)
2023-10-07 16:24:08,190 INFO [server.py:233] connection failed (200 OK)
2023-10-07 16:24:08,190 INFO [server.py:268] connection closed
2023-10-07 16:24:08,190 INFO [server.py:268] connection closed
2023-10-07 16:24:08,190 INFO [server.py:268] connection closed
